text/0000-pack-package-refactor.md

```diff
@@ -0,0 +1,103 @@
+# Meta
+
```

Comment:
Are there any alternative proposals for the structure?

Reply:
Added 👍